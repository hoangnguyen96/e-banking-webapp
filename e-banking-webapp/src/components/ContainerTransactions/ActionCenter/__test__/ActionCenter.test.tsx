// Libs
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';

// Components
import { ActionCenter } from '..';

describe('ActionCenter component', () => {
  test('renders with correct data', () => {
    const { container } = render(<ActionCenter />);

    expect(container).toMatchSnapshot();
  });
});
