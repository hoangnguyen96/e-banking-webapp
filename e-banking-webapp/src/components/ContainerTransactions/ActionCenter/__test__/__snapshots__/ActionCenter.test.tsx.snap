// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActionCenter component renders with correct data 1`] = `<div />`;
